ddtrace/internal/module.py

```diff
@@ -499,7 +457,7 @@ def register_module_hook(cls, module, hook):
         instance = cast(ModuleWatchdog, cls._instance)
         instance._hook_map[module].append(hook)
         try:
-            module_object = instance[module]
+            module_object = sys.modules[module]
```

Comment:
Would this create a strong reference to the module object? Is this an issue?

Reply:
We are not storing the result of this lookup anywhere. So unless the `hook` itself stores the module somewhere (which is out of our control here), we have a net-zero refcount.